addons/udes_sale_stock/models/sale_order.py

```diff
@@ -76,8 +76,9 @@ def init(self):
 
     @api.multi
     def copy(self, default=None):
-        """Append '(copy)' to Customer Reference for duplicated Order"""
```

Comment:
can't we just set the copy flag on the field?

Reply:
This code would always ignore any```client_order_ref ``` that had been specified in ```default```. The idea of the change is to prevent it from overwriting it when a value is passed to it but allow it to continue with the existing behaviour when nothing is passed